chatkit-core/src/main/kotlin/com/pusher/chatkit/subscription/SubscriptionListener.kt

```diff
@@ -0,0 +1,30 @@
+package com.pusher.chatkit.subscription
+
+import com.pusher.platform.SubscriptionListeners
+import elements.EosError
+import elements.Headers
+import elements.SubscriptionEvent
+
+/**
+ * Platform SubscriptionListeners, but as an interface.
+ *
+ * In future, consider pushing this interface down in to platform as an alternative to the
+ * SubscriptionListeners data-class-full-of-closures approach.
```

Comment:
now after simplifications we probably won't be moving it to platfrom, it looks like the whole docs should be gone or rephrased